pandas/tests/indexing/test_multiindex.py

```diff
@@ -1340,3 +1342,20 @@ def test_panel_setitem_with_multiindex(self):
         p5.iloc[0, :, 0] = [1, 2]
         expected = Panel(arr, **axes)
         tm.assert_panel_equal(p5, expected)
+
+
+def test_multiindex_period_datetime():
+    # GH4861, using datetime in period of multiindex raises exception
```

Comment:
did you duplicate this test? it is already in master

Reply:
I didn't explicitly add it again, it was just showing up in the diff for some reason.  I thought I was completely up-to-date with master.  Anyway, I synched again, and I think things are now in a good state because it is no longer showing in the diff.